modules/languages/ruby/ruby.nix

```diff
@@ -0,0 +1,152 @@
+{
+  pkgs,
+  config,
+  lib,
+  ...
+}:
+with lib;
+with builtins; let
+  format-env = pkgs.callPackage ./format_derivation.nix {inherit pkgs;};
```

Comment:
please declare this as a package in flake.packages (see flake/ directory) and then call the `self.packages.<packageName>` attribute

Reply:
a package derivation does not belong in the modules directory